Makefile

```diff
@@ -1,3 +1,4 @@
+BENCH = bench
```

Comment:
The other constants defined here are executable commands. I'd prefer not to define `BENCH`.